### DISCUSSION TITLE
End-user i2pd bundles

### DISCUSSION THREAD

**Discussion by l-n-s** [PurpleI2P/PurpleI2P#12] [Ideas]
i2pd is pretty easy to create bundles with, does not require much external dependencies.  

Because of KISS design i2pd will not have any end-user apps built in, like Java I2P does. But end-user applications are critical for I2P adoption, because installing and configuring i2pd with other apps is hard.

List of currently available end-user applications with i2pd:

* [Browser Bundle](https://github.com/PurpleI2P/i2pdbrowser)
* ...

Related issues: https://github.com/PurpleI2P/i2pd/issues/820

**Comment by majestrate** [comment]
an i2pd bittorrent bundle with [XD](https://github.com/majestrate/XD) could be made, could also think about melding i2pd and XD into same binary but that could get really fuggy really fast.
